API/API.py

```diff
@@ -140,20 +247,39 @@ def is_at_least_one_keyword(category='1'):
     included_keywords = parse_json_and_strip('included_keywords'+category)
     return included_keywords != []
 
-def map_keywords_to_intonation(keywords_list):
+def map_keywords_to_intonation(keywords_list,phrases,positive_keywords,negative_keywords):
     '''
     uses the existing keyword database and queries to map a keyword to the intonation
     (if exists already)
     '''
-    known_keyword_to_intonation=dict(SQLQuery().select_learned_keywords())
+    known_keyword_to_intonation = {t[0]: t[1] for t in SQLQuery().select_learned_keywords()}
+    # known_keyword_to_intonation = SQLQuery().select_learned_keywords()
+    if known_keyword_to_intonation is None:
+        known_keyword_to_intonation={}
+    phrases_no_quotes = [phrase.strip('"\'') for phrase in phrases]
```

Comment:
if you fix the comment at the top to make sure that the quotes are not in the phrase, then there is no need to treat phrases and keywords differently, and there is no need for stripping. therefore there is no need for the changes done here between phrases and keywords

Reply:
i am not sure if we care for duplication, if not then just leave this as is, if yes then make sure phrases has no quotation in the parser function and you can merge keywords and phrases together in the processing phase